### PR TITLE
Colossus's shotgun is now a static-spread blast of 6 bolts

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Colossus's shotgun is now a static-spread blast of 6 bolts, making it more predictable.
/:cl:

Right now, the shotgun's spread is based on how far from you it is; up close, it's a wide spread, and further out it's much more concentrated.
Now it's much more consistent, spread-wise. Also, getting clipped by it is not instantly lethal, though if you're too close when it's fired it probably IS instantly lethal.

###### Also I considered making a spiral shot that fired blasts instead of single shots, with appropriate delay, but it was horrifying and I didn't want to do too much to it.

### CODE REVIEW

**Comment by Repukan** [comment]
Hugbox

**Comment by Repukan** [comment]
If you use anything called shotgun and you change it to not be lethal when you fuck up and refuse to learn to keep a safe distance then what's the point? Might as well change it so instead of damage it heals you then

**Comment by ChangelingRain** [comment]
it's still lethal up close, it's just not a one-shot if you get hit at not-close distances.
you know, following your advice?

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 1]
```diff
@@ -217,12 +182,17 @@ Difficulty: Very Hard
...
-/mob/living/simple_animal/hostile/megafauna/colossus/proc/blast()
-	playsound(get_turf(src), 'sound/magic/clockwork/invoke_general.ogg', 200, 1, 2)
-	var/turf/T = get_turf(target)
-	newtonian_move(get_dir(T, targets_from))
-	for(var/turf/turf in range(1, T))
-		shoot_projectile(turf)
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/blast(set_angle)
+	var/turf/own_turf = get_turf(src)
+	var/turf/target_turf = get_turf(target)
+	playsound(own_turf, 'sound/magic/clockwork/invoke_general.ogg', 200, 1, 2)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: own_turf -> src

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 2]
```diff
@@ -235,9 +205,9 @@ Difficulty: Very Hard
 /mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
 	for(var/mob/M in range(10,src))
 		if(M.client)
-			flash_color(M.client, rgb(200, 0, 0), 1)
+			flash_color(M.client, "#C80000", 1)
 			shake_camera(M, 4, 3)
-	playsound(get_turf(src),'sound/magic/clockwork/narsie_attack.ogg', 200, 1)
+	playsound(get_turf(src), 'sound/magic/clockwork/narsie_attack.ogg', 200, 1)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: get_turf(src) -> src

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 3]
```diff
@@ -235,9 +205,9 @@ Difficulty: Very Hard
 /mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
 	for(var/mob/M in range(10,src))
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: Maybe just livings? But w/e

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: I need own_turf anyway there's no reason to change this one

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm (hunk 3 above)
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: but then ghosts wouldn't get it.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**